modules/openapi-generator/src/main/resources/csharp/libraries/generichost/JsonConverter.mustache

```diff
@@ -577,4 +577,4 @@
             {{/lambda.trimLineBreaks}}
             {{/lambda.trimTrailingWithNewLine}}
         }
-    }
\ No newline at end of file
+    }
```

Comment:
Please remove the spacing changes in this file.

Reply:
Done.